agent/src/main/java/io/pyroscope/api/Tags.java

```diff
@@ -0,0 +1,264 @@
+package io.pyroscope.api;
+
+
+import com.google.gson.annotations.SerializedName;
+import one.profiler.AsyncProfiler;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+import java.util.function.Function;
+
+public class Tags {
```

Comment:
Similar to my comment in the other PR (https://github.com/pyroscope-io/pyroscope/pull/1096/files#r872029411), we should call these labels for the sake of consistency with other profilers.

Reply:
Renamed to 'Labels' and 'LabelsSet'

Reply:
Please let me know if you like new names